src/installation/guides/macbook.md

```diff
@@ -0,0 +1,53 @@
+# Macbookpro 12,1 (early 2015) 
+
+Everything is well supported on this laptop and install will work without problems. Post-installation, after having booted to your newly installed system, there is however a few points that need a little bit of work.
+
+## keymap are wrong
+This is a problem inherited by a few distros, including archlinux.
+tty keymaps for the macbooks (mac-fr, mac-us, mac-de) are completely broken and *you should not attempt to use them*, or it will result in an unusable keyboard and you will be forced to do a hard reset on your system. as a workaround generic keymaps
+work well (fr, us, de ..) you can use those for starter. later on you can find working ones and install/use them. here i've found a working one for the french macbook keyboards: [forums.archlinux.fr](https://forums.archlinux.fr/viewtopic.php?t=8679)
+
+```
+cd /usr/share/kbd/keymaps/mac/all
```

Comment:
Code snippets should follow the style guide: https://docs.voidlinux.org/contributing/void-docs/style-guide.html

Use `$` for unprivileged commands, `#` for privileged commands. In this case, there's also a question of whether the user should be modifying /usr/share or if /usr/local/share is the appropriate place for this.

Reply:
thanks for providing a link to the style guide.
i'll modify the snippets, here i had used # as a comment